src/coreclr/jit/block.cpp

```diff
@@ -617,22 +617,23 @@ void BasicBlock::dspSuccs(Compiler* compiler)
     // compute it ourselves here.
     if (bbKind == BBJ_SWITCH)
     {
-        // Create a set with all the successors. Don't use BlockSet, so we don't need to worry
-        // about the BlockSet epoch.
-        unsigned     bbNumMax = compiler->fgBBNumMax;
-        BitVecTraits bitVecTraits(bbNumMax + 1, compiler);
-        BitVec       uniqueSuccBlocks(BitVecOps::MakeEmpty(&bitVecTraits));
+        unsigned* const targetNums  = new unsigned[NumSucc()];
```

Comment:
```suggestion
        unsigned* const targetNums  = new (compiler, CMK_DebugOnly) unsigned[NumSucc()];
```

But is this change even necessary?

Reply:
I guess not -- if we get to the point where we don't need `bbNum` at all, we can rewrite this logic altogether to print in `bbID` order (or some other order).